charts/merlin/templates/_config.tpl

```diff
@@ -0,0 +1,59 @@
+{{- define "merlin.contextRef" -}}
+{{- $ := index . 0 }}
+{{- $reference := index . 1 }}
+{{ $.Values.imageBuilder.contextRef | default (printf "refs/tags/v%s" $reference) }}
```

Comment:
Just to confirm my understanding, `rendered.releasedVersion` **is not** intended to take precedence over `.Values.imageBuilder.contextRef` from this implementation right?

Reply:
Okay, from the unit tests that seems to be the intention.